rotkehlchen/chain/ethereum/manager.py

```diff
@@ -1195,3 +1195,23 @@ def get_basic_contract_info(self, address: ChecksumEthAddress) -> Dict[str, Any]
 
         self.contract_info_cache[address] = info
         return info
+
+    def _get_basic_contract_info_fallback(
+            self,
+            output: List[Tuple[bool, bytes]],
+            properties: Tuple[str, str, str],
+            contract: EthereumContract,
+    ) -> List:
+        """Acts a fallback for `get_basic_contract_info` when a RemoteError happens.
+        'decimals' property defaults to 18.
+        """
+        decoded_contract_info = []
+        for x, method_name in zip(output, properties):
+            if x[0] is True and len(x[1]) != 0:
+                decoded_contract_info.append(contract.decode(x[1], method_name)[0])
+            else:
+                if method_name == 'decimals':
+                    decoded_contract_info.append(18)
```

Comment:
I believe we don't do it in other places but what about having this as a constant like `DEFAULT_TOKEN_DECIMALS = 18`?

Reply:
yeah.